src/Service.Tests/Configuration/RuntimeConfigLoaderTests.cs

```diff
@@ -30,4 +33,63 @@ public async Task CanLoadStandardConfig(string configPath)
 
         Assert.IsTrue(loader.TryLoadConfig("dab-config.json", out RuntimeConfig _), "Failed to load config");
     }
+
+    [DataTestMethod]
+    [DataRow("Multidab-config.CosmosDb_NoSql.json", new string[] { "Multidab-config.MsSql.json", "Multidab-config.MySql.json", "Multidab-config.PostgreSql.json" })]
+    public async Task CanLoadValidMultiSourceConfig(string configPath, IEnumerable<string> dataSourceFiles)
```

Comment:
Can you add a test description that notes that this test only checks that the runtimeconfig was parsed with the expected file names present in the deserialized RuntimeConfig object?
e.g. this test does not validate that all 4 files actually exist in the file system. (as that check is performed elsewhere, perhaps in RuntimeConfigValidator class.)

Reply:
Added description.